app/components/footer_component.html.erb

```diff
@@ -38,19 +38,14 @@
         </a>
       </div>
       <div class="site-footer-top__links-container">
-        <% helpers.navigation_resources.each do |resource| %>
-          <a href="<%= resource.path %>"><%= resource.title %></a>
-        <% end %>
-        <%= link_to "Find an event near you", events_path %>
-        <%= link_to "Sign up for updates", "/mailinglist/signup/name" %>
-        <%= link_to("Search", search_path) %>
       </div>
     </div>
     <div class="site-footer-bottom">
       <div class="site-footer-bottom__logo-container">
         <%= image_pack_tag 'static/dfelogo-white.svg', alt: "Department for Education", size: "124x73" %>
```

Comment:
When using a screen reader on VoiceOver, the focus order feels wrong - the DfE logo alt text is read out between the YouTube link and the Search link

(It is not tabbed to during focus order, just read out by screen readers)

I'm not sure whether you can do anything about this?

Reply:
I don't really understand why that would be the case as the code is clearly in a logical order. I'll take a look into this today.